tools/wptrunner/wptrunner/executors/executorselenium.py

```diff
@@ -187,6 +187,9 @@ def send_message(self, message_type, status, message=None):
             obj["message"] = str(message)
         self.webdriver.execute_script("window.postMessage(%s, '*')" % json.dumps(obj))
 
+def _isAndroidCapabilities(capabilities):
```

Comment:
I think in general reftests don't (or at least didn't) really work on Android. Last time I tried ChromeDriver wasn't able to take screenshots from Chrome for Android. Besides, even if we are able to take screenshots, most reftests are written with the assumption that the screen is bigger than a certain size (800x600) and on small mobile screens important parts of the tests might be out of the viewport.

Reply:
Hmm.. How about running the tests on a certain model of tablet using emulator?

BTW, could you give me an example of reftest?

Reply:
FWIW Gecko has some magic where it ends up taking a screenshot on a viewport that's not fixed to the actual dimensions of the device.

Also note that reftests tend to require 1dppx to work correctly, so it's highly likely that modern phones do have enough physical pixels, but are usually operating at at least 2dppx.

Reply:
Perhaps the most basic reftest: `infrastructure/reftest/green-ref.html`

Reply:
I don't think I can fix reftests in this PR. The goal is to make this minimal viable implementation.
PTAL

Reply:
I think this change is probably hidden a bit too deep for a workaround. We should probably skip reftests using the command line flag (`--test-types wdspec,testharness`, i.e. exclude `reftest`), then `SeleniumRefTestExecutor` won't be exercised and the runner won't try to set the window size.

Reply:
Good idea. Removed for now.